include/ada/character_sets-inl.h

```diff
@@ -441,6 +441,138 @@ constexpr uint8_t PATH_PERCENT_ENCODE[32] = {
     // F8     F9     FA     FB     FC     FD     FE     FF
     0x01 | 0x02 | 0x04 | 0x08 | 0x10 | 0x20 | 0x40 | 0x80};
 
+constexpr uint8_t COMPONENT_PERCENT_ENCODE[32] = {
```

Comment:
Can you add reference links to the spec?

Reply:
I tried to follow the layout of the other tables. Also, if you pay attention, it is self-documented!!! :-) Look carefully. :-)